File: src/main/java/cn/xunyard/idea/coding/log/Logger.java
package cn.xunyard.idea.coding.log;

/**
 * @author <a herf="mailto:wuqi@terminus.io">xunyard</a>
 * @date 2019-12-26
 */
public interface Logger {

    void clear();

    void debug(String format, Object... args);

    void info(String format, Object... args);

    void warn(String format, Object... args);

    void error(String format, Object... args);

    void done();
}

File: src/main/kotlin/cn/xunyard/idea/coding/log/Logger.kt
package cn.xunyard.idea.coding.log

/**
 * @author <a herf="mailto:xunyard@gmail.com">xunyard</a>
 * @date 2019-12-26
 */
interface Logger {
    fun clear()

    fun debug(format: String, vararg args: Any)

    fun info(format: String, vararg args: Any)

    fun warn(format: String, vararg args: Any)

    fun error(format: String, vararg args: Any)

    fun done()
}